js_parser/extract_es_grammar.py

```diff
@@ -20,24 +20,169 @@
 
     make js_parser/es.esgrammar
 
+You can also use this script on a random HTTPS URL, like:
+
+    python extract_esgrammar.py https://raw.githubusercontent.com/tc39/proposal-class-fields/master/spec.html
+
 """
 
+import argparse
+import urllib
 import html5lib
+import re
 from textwrap import dedent
+import xml.etree.ElementTree as ET
```

Comment:
this seems to be unused

Reply:
Oh, right. I was using `ET.dump` while debugging this. Deleted.